test/mode/modeVisual.test.ts

```diff
@@ -810,4 +810,136 @@ suite("Mode Visual", () => {
     });
 
   });
-});
+
+  suite("search works in visual mode", () => {
```

Comment:
I just copied the current file from master in the main repo and rewrote the test. I thought they were added by you guys. I they are not needed i can delete them.